humanlayer-wui/src/components/internal/SessionDetail/SessionDetail.tsx

```diff
@@ -611,8 +611,29 @@ function SessionDetail({ session, onClose }: SessionDetailProps) {
     try {
       setIsLaunchingDraft(true)
 
-      // Get the prompt text from the editor
-      const prompt = responseEditor?.getText() || ''
```

Comment:
Making an assumption here that `getText()` was not working as expected once we introduced slash commands?